product_replenishment_cost/README.rst

```diff
@@ -2,9 +2,9 @@
    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
    :alt: License: AGPL-3
 
-==========================
+--------------------------
```

Comment:
FYI information, new OCA tools provide autogeneration of readme.rst files, based on rst files present in readme folder. 
see : 
https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

Once done, the changes are more easy. (not a blocking point)